setup.py

```diff
@@ -36,21 +37,27 @@ def read(*rnames):
           + '\n\n' +
           read('CHANGES.txt')
           ),
-      keywords = "zope3 skin zmi",
-      classifiers = [
+      keywords="zope3 skin zmi",
+      classifiers=[
           'Development Status :: 5 - Production/Stable',
           'Environment :: Web Environment',
           'Intended Audience :: Developers',
           'License :: OSI Approved :: Zope Public License',
           'Programming Language :: Python',
+          'Programming Language :: Python:: 2.7',
+          'Programming Language :: Python:: 3.5',
```

Comment:
Per the [Python devguide](https://docs.python.org/devguide/#status-of-python-branches), Python 3.4 is not EOL until 2019-03-16.  Shouldn't we be supporting it?  Python 3.3. goes EOL 2017-09-29, which makes it more obvious to drop / avoid.